lib/rubocop/cop/style/nil_comparison.rb

```diff
@@ -41,7 +41,7 @@ class NilComparison < Base
         def_node_matcher :nil_comparison?, '(send _ {:== :===} nil)'
 
         # @!method nil_check?(node)
-        def_node_matcher :nil_check?, '(send _ :nil?)'
+        def_node_matcher :nil_check?, '(send ^_ :nil?)'
 
         def on_send(node)
           style_check?(node) do
```

Comment:
I think it might be clearer to write `return unless node.receiver` at the beginning of `def on_send` rather than using `^`.

```suggestion
        def_node_matcher :nil_check?, '(send _ :nil?)'

        def on_send(node)
          return unless node.receiver

          style_check?(node) do
```

Reply:
👍